demos/general/demo_pipeline_cnmfE.py

```diff
@@ -76,7 +76,7 @@ def main():
     #                      change this one if algorithm does not work
     max_shifts = (5, 5)  # maximum allowed rigid shift
     strides = (48, 48)   # start a new patch for pw-rigid motion correction every x pixels
-    overlaps = (24, 24)  # overlap between pathes (size of patch strides+overlaps)
+    overlaps = (24, 24)  # overlap between paths (size of patch strides+overlaps)
```

Comment:
The original was a misspelling of "patches", not "paths"

Reply:
cool -- will change all the duplicate hits this way